README.md

```diff
@@ -308,7 +308,7 @@ a blog post about the good stuff. You can read more about this [on my
 blog](https://alexschroeder.ch/wiki/2017-04-27_Record_Keeping).
 
 **Alternatives**: Check out [forget](https://forget.codl.fr/about/),
-which is a web app that only expires your toots without archiving the.
+which is a web app that only expires your toots without archiving them.
 Depending on your needs, that might be enough?
```

Comment:
I am also doubting the question mark...

Reply:
I think it works and I'd like the documentation to be more informal, so I'm hoping that helps? Haha.